server/blob/blob-objectstorage/src/main/java/org/apache/james/blob/objectstorage/ObjectStorageBlobsDAO.java

```diff
@@ -67,6 +69,22 @@ public static ObjectStorageBlobsDAOBuilder builder(SwiftKeystone3ObjectStorage.C
         return SwiftKeystone3ObjectStorage.daoBuilder(testConfig);
     }
 
+    public CompletableFuture<ContainerName> createContainer(ContainerName name) {
```

Comment:
I have the impression that you synchronously execute the container creation then wrap it in a future.

Is thread sychronisation really needed here?

Reply:
it is not, I have updated accordingly